geoids/geoid_intro.md

```diff
@@ -0,0 +1,60 @@
+# What is a "geoid"?
+
+A geoid is a model of Earth's shape representing the global mean sea level extended beneath continents. It’s not a smooth, symmetrical ellipsoid but an undulating equipotential surface of gravity shaped by variations in Earth’s gravitational field (e.g., due to mountains, trenches, rock density differences). The geoid serves as the reference for "height above sea level" in many mapping, engineering, and scientific applications: when you get a GPS (ellipsoidal) height, you subtract the geoid height to estimate a more intuitive elevation. Understanding the geoid is crucial because it underpins how we interpret elevations, design infrastructure, model floods, and monitor Earth processes.
+
+* The geoid approximates where global sea level would lie under land, driven by gravity rather than waves or tides. While we can’t place a physical marker on the geoid everywhere, models let us estimate its shape accurately.
```

Comment:
```suggestion
* The geoid approximates where global sea level would lie under land, as determined by Earth's gravity rather than waves or tidal forces. While we can’t place a physical marker on the geoid everywhere, models let us estimate its shape accurately.
```